Royalty/DefInjected/ResearchProjectDef/ResearchProjects_Implants.xml

```diff
@@ -2,58 +2,58 @@
 <LanguageData>
   
   <!-- EN: artificial metabolism -->
-  <ArtificialMetabolism.label>TODO</ArtificialMetabolism.label>
+  <ArtificialMetabolism.label>Sztuczny metabolizm</ArtificialMetabolism.label>
   <!-- EN: Craft bionics that process food more safely or efficiently than a biological stomach. -->
-  <ArtificialMetabolism.description>TODO</ArtificialMetabolism.description>
+  <ArtificialMetabolism.description>Stwórz modyfikacje bioniczne które trawią jedzenie bezpieczniej i efektywniej niż biologiczny żołądek.</ArtificialMetabolism.description>
   
   <!-- EN: brain wiring -->
-  <BrainWiring.label>TODO</BrainWiring.label>
+  <BrainWiring.label>Okablowanie mózgu</BrainWiring.label>
   <!-- EN: Craft brain implants that induce or prevent blunt sensations like joy or pain. -->
-  <BrainWiring.description>TODO</BrainWiring.description>
+  <BrainWiring.description>Stwórz implanty mózgu które wzbudzają bądź powstrzymują uczucia takie jak radość czy ból.</BrainWiring.description>
   
   <!-- EN: circadian influence -->
-  <CircadianInfluence.label>TODO</CircadianInfluence.label>
+  <CircadianInfluence.label>Cykl okołodobowy</CircadianInfluence.label>
   <!-- EN: Craft brain implants that chemically manipulate the circadian rhythm, reducing or altering the need to sleep. -->
-  <CircadianInfluence.description>TODO</CircadianInfluence.description>
+  <CircadianInfluence.description>Stwórz implanty mózgu które chemicznie manipulują rytmem okołodobowym, zmniejszając bądź zmieniając potrzebę snu.</CircadianInfluence.description>
   
   <!-- EN: compact weaponry -->
-  <CompactWeaponry.label>TODO</CompactWeaponry.label>
+  <CompactWeaponry.label>Broń kompaktowa</CompactWeaponry.label>
   <!-- EN: Craft compact, concealable bionic weapons which can be embedded in various body parts. -->
-  <CompactWeaponry.description>TODO</CompactWeaponry.description>
+  <CompactWeaponry.description>Stwórz kompaktową, łatwą do ukrycia broń która może być schowana w różnych częściach ciała.</CompactWeaponry.description>
   
   <!-- EN: flesh shaping -->
-  <FleshShaping.label>TODO</FleshShaping.label>
+  <FleshShaping.label>Kształtowanie ciała</FleshShaping.label>
   <!-- EN: Craft bionics which induce the reshaping or natural flesh, usually for the purposes of aesthetic enhancement or personal pleasure. -->
-  <FleshShaping.description>TODO</FleshShaping.description>
+  <FleshShaping.description>Stwórz modyfikacje bioniczne wspomagające przekształtowanie skóry, przeważnie stosowane w wzbogaceniach estetycznych bądź dla przyjemności.</FleshShaping.description>
```

Comment:
Piszę jeszcze raz, bo chyba brakło tych zmian:
Przekształtowanie -> przekształcanie
stosowane **we** wzbogaceniach
Zmień proszę tutaj też ze skóry na ciała.

ps. da się jakoś zrobić unmark na konwersacji?

Reply:
Racja, moja wina, pominąłem tą linijke przez przypadek. 🤦‍♂️ 

Unmark może dać twórca PR, czyli w tym przypadku ja.